client/src/main/scala/drt/client/components/TerminalComponent.scala

```diff
@@ -283,9 +280,9 @@ object TerminalComponent {
         <.li(^.className := tabClass(Staffing),
           props.router.link(props.terminalPageTab.update(
             mode = Staffing,
-            subMode = "15",
+            subMode = "60",
```

Comment:
you mean if they edit in hourly it would override quater values specially port like lhr does it . May be good point let me check with Rob or in meeting

Reply:
Yeah we already agreed last week we can't change the default view